### ISSUE TITLE
opentelemetry-kube-stack production maturity 

### ISSUE THREAD

**Issue by genadipost** [open-telemetry/opentelemetry-helm-charts#1399]
Hello all!

I am considering replacing Prometheus/Vector stack with [opentelemetry-kube-stack](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack) from k8s metric and logs gathering.
Except the fact the stability status of the chart is **alpah** (which is a lesser concern for me) i noticed the following description in the readme:

> This Helm chart serves as a quickstart for OpenTelemetry in a Kubernetes environment. 

Which makes me wonder if this chart should only be used for POC and testings proposes and if i want to deploy OTEL in production i should the "vanilla" [opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator) and copy the configurations generated in opentelemetry-kube-stack.

**Comment by jaronoff97** [comment]
@genadipost I am currently using the chart in production (with a few modifications for things like node selectors, affinities, resource requests, etc.) without issues! There's definitely going to be a few hiccups along the way, but I think it serves as a great starting place!

**Comment by TylerHelmuth** [comment]
I agree that the chart is ready to use, but the alpha warning is still appropriate because it is so new.
